mypy/build.py

```diff
@@ -2421,6 +2415,25 @@ def find_module_and_diagnose(manager: BuildManager,
             raise CompileError(["mypy: can't find module '%s'" % id])
 
 
+def find_added_packages(suppressed: List[str],
+                        manager: BuildManager, options: Options) -> bool:
+    """Find if there are any newly added packages that were previously suppressed.
+
+    Exclude everything not in build for follow-imports=skip.
+    """
+    for dep in suppressed:
+        path = manager.find_module_cache.find_module(dep, manager.search_paths,
+                                                     manager.options.python_executable)
+        if not path or dep in manager.source_set.source_modules:
+            continue
+        if (options.follow_imports == 'skip' and
+                (not path.endswith('.pyi') or options.follow_imports_for_stubs)):
+            continue
+        if '__init__.py' in path:
```

Comment:
This is not 100% correct -- the path could be like `foo/foo__init__.py`, for example (or there could even be a directory named `__init__.py` in the path, though this would almost never happen).

Reply:
Yes, but this doesn't affect correctness, only performance. Anyway, I will try co find a more precise test.